Parse/Parse/Internal/Commands/CommandRunner/URLSession/PFURLSessionCommandRunner.m

```diff
@@ -261,7 +262,8 @@ - (BFTask *)_performCommandRunningBlock:(nonnull id (^)(void))block
 
 + (NSURLSessionConfiguration *)_urlSessionConfigurationForApplicationId:(NSString *)applicationId
                                                               clientKey:(nullable NSString *)clientKey {
-    NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
+    NSURLSessionConfiguration *customConfiguration = Parse._currentManager.configuration.URLSessionConfiguration;
+    NSURLSessionConfiguration *configuration = customConfiguration ?: [NSURLSessionConfiguration defaultSessionConfiguration];
```

Comment:
the `URLSessionConfiguration` should always be set on the configuration, so tghis way this is not required